<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="QT-INBG-LT0591" failures="10" tests="10" name="Default test(failed)" time="60.168" errors="0" timestamp="2022-07-15T12:45:05 IST">
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="5.288">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="7.052">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="4.608">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="6.139">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="6.065">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="5.809">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="5.389">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="5.879">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="7.198">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderInvalidTest" time="6.665">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Automation.Mobile.OrderTest.orderInvalidTest(OrderTest.java:112)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- orderInvalidTest -->
  <testcase classname="Automation.Mobile.OrderTest" name="orderTest" time="0.0">
    <ignored/>
  </testcase> <!-- orderTest -->
</testsuite> <!-- Default test(failed) -->
